docs/aspnetcore-facility.md

```diff
@@ -127,30 +208,39 @@ public sealed class CustomMiddleware : IMiddleware
 }
 ```
 
-## Registering ASP.NET Core framework components in Windsor
-
-Simply put, don't do this. The instance that Windsor resolves might not always be the same instance that is resolved by the ASP.NET Core framework. If the 
-instance is stateful or is registered with a competing lifestyle this can lead to problems. 
-
 ### Torn lifestyles
 
-If you register a framework component as scoped in Windsor and the ASP.NET Core framework has registered it as scoped also, two instances might appear. This
-problem is further obfiscated if those objects are stateful. This is known as a `torn lifestyle`. 
+If you register an application component such as a singleton in Windsor and ASP.NET Core framework two instances might appear. This 
+is known as a `torn lifestyle`. We hope to have avoided this in this facility's design but it is useful to know what symptoms to look for
+if this does happen.
+
+Please report any evidence of this on our issue tracker.
 
 References:
  - https://simpleinjector.readthedocs.io/en/latest/tornlifestyle.html
 
 ### Captive Dependencies
 
-This is when Windsor services with `long lived`(ie. Singleton) lifestyles, consume framework components that were registered with `short lived`(ie. Transient, Scoped) lifestyles. 
-The framework service dependency is effectively held captive by it's consumer lifestyle. These `side effects` can be hard to diagnose. 
+This is when Windsor services with `long lived`(ie. Singleton) lifestyles, consume components that were registered with `short lived`(ie. Transient, Scoped) 
+lifestyles. The service dependency is effectively held captive by it's consumer lifestyle. Symptoms here could include dispose method's not firing for
+transients and scopes that are consumed by singletons. 
 
 References:
  - http://blog.ploeh.dk/2014/06/02/captive-dependency/
 
-Special credit goes to:
+### Typed Factories
+
+Typed factories are supported although there are some problems we do know about. 
+
+```csharp
+Container.Register(Component.For<LateBoundTransientDisposable>().UsingFactoryMethod(() => new LateBoundTransientDisposable()).LifestyleTransient().CrossWired());
+```
+
+After rigourous testing we noticed a small memory leak, we are into typed factories here: https://github.com/castleproject/Windsor/issues/410
+
```

Comment:
@jonorossi && @generik0 

Small memory leak here. Just thought we would call that out. The process creeps up at a glacial pace. We are talking Kb's every hour or so(framework??). Either way, if people are picking this up and using typed factories they should at least have a warning of the stuff we know about.